src/main/java/nl/nn/testtool/web/AngularServlet.java

```diff
@@ -132,11 +132,12 @@ public ServletOutputStream getOutputStream() throws IOException {
 			};
 		}
 		final String webJarsBase = "/webjars/";
+		final String angular_18_directory_browser = "/browser";
```

Comment:
As I understand from https://github.com/angular/angular-cli/issues/26304 there is a browser and (possibly) a server folder and other resources too. Only the files in the browser folder (the "publicly accessible files") are relevant for our frontend jar. I think the frontend build system will now include the server folder and other resources too (when present). I think we should change the pom.xml of the frontend to only include the files in the browser folder (without the browser folder itself). Basically change https://github.com/wearefrank/ladybug-frontend/blob/df54aacf2c5d65509d5dd5f1b1bdc52815745130/pom.xml#L15 to ${basedir}/dist/browser. But I think we should also split this property into two properties:

    <compiled.project.location>${basedir}/dist</compiled.project.location>
    <compiled.frontend.location>${compiled.project.location}/dist/ladybug/browser</compiled.frontend.location>

And use the first one at https://github.com/wearefrank/ladybug-frontend/blob/df54aacf2c5d65509d5dd5f1b1bdc52815745130/pom.xml#L31 and remove /ladybug from https://github.com/wearefrank/ladybug-frontend/blob/df54aacf2c5d65509d5dd5f1b1bdc52815745130/pom.xml#L104C59-L104C67 and https://github.com/wearefrank/ladybug-frontend/blob/df54aacf2c5d65509d5dd5f1b1bdc52815745130/pom.xml#L123

Reply:
I created a [PR on the frontend](https://github.com/wearefrank/ladybug-frontend/pull/532) to fix this